src/ds323x/alarms.rs

```diff
@@ -157,19 +197,40 @@ where
         when: DayAlarm2,
         matching: Alarm2Matching,
     ) -> Result<(), Error<CommE, PinE>> {
```

Comment:
Same as above: Could you document here how the values of invalid inputs are adapted when these are ignored by the selected matching strategy?